### PR TITLE
(DOCSP-23437): Update free tier threshold billing description

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

This PR clarifies when App Services starts billing for usage and what that means.

I also added an h4 for "Free tier thresholds" to make it super clear where readers can find that info.

Also also, I noticed what seems to be a CSS issue in the free tier thresholds section (not introduced or solved in this PR).

Screen Shot 2022-10-24 at 9 01 27 AM

There's no space between the bolded "or"s and the refs that precede them. Just noting it here. I'll look into whether this is a known issue or not.

EDIT: That CSS issue is being addressed by Ben in https://github.com/mongodb/docs-app-services/pull/194

### Jira

- https://jira.mongodb.org/browse/DOCSP-23437

### Staged Changes (Requires MongoDB Corp SSO)

- [Billing](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-23437/billing/#monthly-free-tier)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/billing.txt [hunk 1]
```diff
@@ -44,9 +44,11 @@ Monthly Free Tier
 
 All App Services apps in a MongoDB Atlas project share a single monthly free
 tier. All usage below the free tier thresholds in a given month is not billed.
-Once the sum of all application usage in a project exceeds a free tier
-usage threshold, App Services immediately begins to bill for any additional
-usage.
+As soon as any monthly free tier threshold is exceeded, App Services starts 
+billing for additional usage of any kind.
```
[the remark this answers is not in the document]
Comment (krollins-mdb) [reply source/billing.txt:48]: Hmm. On line 45, we say "All App Services apps in a MongoDB Atlas project share a single monthly free tier". Perhaps this is my lack of product knowledge, but that seems to suggest that free tier thresholds are at the project level, not the app level.

If this isn't the case, then we should modify line 45.

[the remark this answers is not in the document]
Comment (mongodben) [reply source/billing.txt:48]: >  but that seems to suggest that free tier thresholds are at the project level, not the app level.

can you talk to an SME (maybe ask in the #ask-app-services) channel to confirm that it's project level, and update the documentation to specify.

**Comment by mongodben** [comment]
also your note about the weird formatting with bold text after link was addressed by https://github.com/mongodb/docs-app-services/pull/194

File: source/billing.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (krollins-mdb) [reply source/billing.txt]: @mongodben, Drew confirmed the thresholds are at the project level. I've updated to use your suggested active voice, but subbed in "project" for "App".

**Review by mongodben — APPROVED** [review]
LGTM! might also want to quickly run this PR by Drew to make sure 100% accurate.

**Pull request merged**